documentation/guides/migrating-from-v8-to-v9.md

```diff
@@ -75,6 +75,31 @@ CSS custom properties that have been deprecated can be replaced with the CSS pro
 | `--p-shadow-from-direct-light`  |  `rgba(0, 0, 0, 0.15)`   |
 |   `--p-shadow-from-dim-light`   |   `rgba(0, 0, 0, 0.2)`   |
 
+## Sass functions and mixins
+
+The following sass functions and mixins have been removed. You will either need to add the functions to your repo or replace all function instances with values.
+
+### Adding the functions and mixins to your repo
+
+To help you quickly add these functions and mixins back to your repo, we've created a css file with all the removed functions and mixins.
+
+[✨ Amazing mega file linked here ✨]
+
+### Replacing function and mixin instances with values or tokens
+
+To help you replace function and mixin instances in your repo, we've provided the following list of functions/mixins and their value equivalents or new token values.
+
+#### `easing()`
+
+| Function                 | Replacement Value/Token              |
+| ------------------------ | ------------------------------------ |
+| easing()<br>easing(base) | var(--p-ease)                        |
```

Comment:
```suggestion
| `easing()`<br>`easing(base)` | `var(--p-ease)`                        |
```

Reply:
This is just a suggestion, not sure if we want to do this for all of them.

Reply:
Good call! Yeah I think it makes sense to do it for all of them 👍🏻